build-browser.js

```diff
@@ -15,9 +15,9 @@ rollup({
   ],
 }).then(bundle => (
   bundle.write({
-    dest: 'validator.js',
+    file: 'validator.js',
     format: 'umd',
-    moduleName: pkg.name,
+    name: pkg.name,
```

Comment:
Same to the changes here.

Reply:
Same, it has been replaced [output](https://rollupjs.org/guide/en/#output-name)